### PR TITLE
Parser for Proxyspy

### PR DESCRIPTION
**Pull request by jgedeon120**
If #800 is approved here is the Proxyspy parser that uses the classification.type proxy and the taxonomy of Suspicious.

### CODE REVIEW

**Comment by sebix** [comment]
@aaronkaplan Please review the classification part.

File: intelmq/bots/BOTS [hunk 1]
```diff
@@ -378,6 +378,11 @@
             "module": "intelmq.bots.parsers.phishtank.parser",
             "parameters": {}
         },
+        "Pproxyspy": {
```
Comment (SYNchroACK) [inline intelmq/bots/BOTS]: small thing: there is an extra 'P'

Comment (SYNchroACK) [inline intelmq/bots/BOTS]: small thing: there is an extra 'P'

File: docs/Data-Harmonization.md [hunk 2]
```diff
@@ -146,6 +146,7 @@ The taxonomy can be automatically added by the taxonomy expert bot based on the
 |ddos|Availability|This IOC refers to various parts of the DDOS infrastructure.|
 |dropzone|Information Content Security|This IOC refers to place where the compromized machines store the stolen user data.|
 |phishing|Fraud|This IOC most often refers to a URL, which is phishing for user credentials.|
+|proxy|Suspicious|This refers to the use of proxies from inside your network.|
```
Comment (SYNchroACK) [inline docs/Data-Harmonization.md]: According to eCSIRT, `Suspicious` needs to be replaced by `Other`. @aaronkaplan double-check please.

**Comment by SYNchroACK** [comment]
@jgedeon120 ping

As soon as possible, check the changes requested and if possible fix the conflict. I really want to merge this one :D

File: intelmq/bots/experts/taxonomy/expert.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 TAXONOMY = {
     "phishing": "Fraud",
+    "proxy": "Suspicious",
```
Comment (SYNchroACK) [inline intelmq/bots/experts/taxonomy/expert.py]: sed 's/Suspicious/other/g'

**Comment by SYNchroACK** [comment]
> @aaronkaplan Please review the classification part.

Reviewed on #800 issue. The value for `classification.taxnomy` is `other`

**Comment by wagner-certat** [comment]
Rebased, fixed the suspicious/other bug and merged.

Thanks!

File: docs/Data-Harmonization.md (hunk 2 above)
Comment (aaronkaplan) [inline docs/Data-Harmonization.md]: +1

File: intelmq/bots/experts/taxonomy/expert.py (hunk 3 above)
Comment (aaronkaplan) [inline intelmq/bots/experts/taxonomy/expert.py]: +1. pls change

File: intelmq/bots/parsers/proxyspy/parser.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        else:
+            value = line.split(' ')
+            self.new_event(report)
+            event.add('time.source', self.lastgenerated)
+            event.add('source.ip', value[0].split(':')[0])
+            event.add('source.port', value[0].split(':')[1])
+            event.add('source.geolocation.cc', value[1].split('-')[0])
+            event.add('event_description.text', 'Possible HTTP/HTTPS proxy usage when IP and port match for '
+                                                'outbound traffic.')
+            event.add('classification.type', 'proxy')
```
Comment (aaronkaplan) [inline intelmq/bots/parsers/proxyspy/parser.py]: would it make sense to call the type "openproxy" ?

**Comment by aaronkaplan** [comment]
it's a great addition! Thank you!!!!
Please merge if not already done.

**Comment by sebix** [comment]
> it's a great addition! Thank you!!!!
Please merge if not already done.

It's already merged and the taxonomy has been fixed too.

**Pull request closed without merging**